integr8ly/tutorial-web-app
# add rhsso default service

Adds a link to RH-SSO to the applications list

Verification steps:

This and https://github.com/integr8ly/managed-service-broker/pull/37 are deployed to https://master.pbraun-73b1.openshiftworkshop.com/console/catalog

1. Login to the webapp on this cluster using an evals user (evals12 is taken)
1. Wait until all services are provisioned (there should be an entry for Red Hat Single Sign-On in the list)
1. Make sure that the application list on the overview page has an entry for Red Hat Single Sign-On
1. Make sure that the link is correct
